File: exercises/small_problems/easy9/counting_up.rb
# GIVEN an integer
# RETURN array of all integers starting with 1..int

def sequence(int)
  return [0] if int == 0
  int > 0 ? (1..int).to_a : (int..-1).to_a
end

p sequence(5) == [1, 2, 3, 4, 5]
p sequence(3) == [1, 2, 3]
p sequence(1) == [1]

File: exercises/jsSmallProblems/easy5/countingUp.js
const sequence = (num) => [...new Array(num)].map((_, i) => i + 1);

console.log(sequence(5)); // [1, 2, 3, 4, 5]
console.log(sequence(3)); // [1, 2, 3]
console.log(sequence(1)); // [1]
